Runtime/Helpers/CinemachineDefaultMouseInput.cs

```diff
@@ -0,0 +1,75 @@
+#if CINEMACHINE_UNITY_INPUTSYSTEM
+
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace Cinemachine
+{
+    /// <summary>
+    /// Static class that returns a reference to InputSystem package's PlayerLook input asset.
+    /// </summary>
+    static class CinemachineDefaultMouseInput
+    {
+        static InputActionReference s_InputActionReference = null;
+        static CinemachineDefaultMouseInput()
+        {
+            var inputActionAsset = AssetDatabase.LoadAssetAtPath<InputActionAsset>("Packages/com.unity.inputsystem/" +
+                "InputSystem/Plugins/PlayerInput/DefaultInputActions.inputactions");
+
+            InputAction look;
+            var enumerator = inputActionAsset.GetEnumerator();
+            for (int i = 0; i < 40; ++i)
+            {
+                if (enumerator.Current != null && 
+                    enumerator.Current.ToString() == "Player/Look[/Mouse/delta,/Pen/delta]")
+                {
+                    look = enumerator.Current;
+                    s_InputActionReference = InputActionReference.Create(look);
+                    s_InputActionReference.name = "Generic Look";
+                    break;
+                }
+                enumerator.MoveNext();
+            }
+        }
+        static InputActionReference GetInputActionReference()
+        {
+            return s_InputActionReference;
+        }
+
+        static GUIContent s_InputProviderAddLabel = new GUIContent(
+            "Add CinemachineInputProvider", "Adds CinemachineInputProvider to this vcam, if it does not have one already, " +
+            "enabling the vcam to read input from Input Actions. By default, a simple mouse XY input action is added.");
+        
+        /// <summary>
+        /// Adds a button that adds adds CinemachineInputProvider component to the vcam with a default look control
+        /// (XY axis), if the gameobject has at least one component or extension that requires input and the vcam does
+        /// not already have a CinemachineInputProvider component. For a component or extension to require input,
+        /// the component or extension needs to override RequiresInput in CinemachineComponentBase or
+        /// CinemachineExtension respectively.
+        /// <seealso cref="CinemachineVirtualCameraBaseEditor"/>
+        /// </summary>
+        /// <param name="gameObject">The gameObject to which we'd like to add the CinemachineInputProvider
+        /// via a Button interface</param>
+        public static void InputProviderButton(GameObject gameObject)
```

Comment:
This is not too pretty when the inspector is small

![image](https://user-images.githubusercontent.com/6424658/112470228-cd7ee180-8d40-11eb-80bf-636537ca9941.png)

Reply:
I agree. Asked Julien last night, what his suggestion is for this case.